### PR TITLE
Add Ramp<>

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

File: functions/ramp.h [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#ifndef FUNCTIONS_RAMP_H
+#define FUNCTIONS_RAMP_H
+
+//Returns led as value between 0 ~ 32768
+
+class Ramp {
```
Comment (profezzorn) [inline functions/ramp.h:6]: Actually, when I think about it, I think we should call it RampF<>
It makes it easy to see that it's a function, and it's much less likely to collide with something.

Comment (fdarosa2663) [reply functions/ramp.h]: updated

File: functions/ramp.h [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#ifndef FUNCTIONS_RAMP_H
+#define FUNCTIONS_RAMP_H
+
+//Returns led as value between 0 ~ 32768
```
Comment (profezzorn) [inline functions/ramp.h:4]: Needs a better description/comment.

Comment (fdarosa2663) [reply functions/ramp.h]: added with use example

**Pull request merged**